### DISCUSSION TITLE
Transcription performance of different GPU's

### DISCUSSION THREAD

**Discussion by cetiny** [openai/whisper#480] [Q&A] [answered]
I plan to use Whisper to get several podcast episodes transcribed using my desktop PC. I would like to use the "medium.en" model to get the optimum quality. I also have the beam_size = 5, which improves the accuracy according to my tests.

I currently own a GTX 1070Ti GPU. A one hour audio file takes around 30 minutes to transcribe. 

I plan to purchase a newer GPU, potentially a 3080, 3090 or 4090. 

Can you share your experience with GPU's please, so that I can make an informed choice? Thank you!

**Comment by Tortoise17** [comment, marked as answer]
Principally and physically on ground reality I have performed some tests starting from smaller GPU until biggest possible GPU.  large model works without problem from NVIDIA RTX 2080TI and above. This is what I have tested so far. If you are planing to use big model, better take 16 GB + GPU (16GB and above for easy use).  It needs more than 12GB ram for single file while transcribing. In principle 4090 is ideal, 3090TI should also work good without problems and it is available as well as cost is low. Regarding RAM requirements, 32 GB  RAM  minimum plus 5 GB additional for each single file if you are transcribing parallel more than 1 file.

**Reply by cetiny** [reply]
Thank you! Is there a significant performance improvement between 4090 and 3090Ti? 4090 is very hard to get right now in Germany unfortunately.

**Reply by Tortoise17** [reply]
@cetiny for all GPUs, there is long waiting in Germany. At the moment, you can buy with higher prices. When I replied you, that exact time the GPU was available. I guess better wait a little to get good device which runs with you for long time. I have personally not tested 4090. I have tested on quadro 8000, RTX 2080TI and 1650.

**Reply by cetiny** [reply]
Thank you @Tortoise17 . Yes, I guess I will wait for a while until the GPU market normalizes.

**Reply by arabcoders** [reply]
> Principally and physically on ground reality I have performed some tests starting from smaller GPU until biggest possible GPU. large model works without problem from NVIDIA RTX 2080TI and above. This is what I have tested so far. If you are planing to use big model, better take 16 GB + GPU (16GB and above for easy use). It needs more than 12GB ram for single file while transcribing. In principle 4090 is ideal, 3090TI should also work good without problems and it is available as well as cost is low. Regarding RAM requirements, 32 GB RAM minimum plus 5 GB additional for each single file if you are transcribing parallel more than 1 file.

Hi, do you have tips on how to run parallel transcribe using the same loaded model?

**Comment by FurkanGozukara** [comment]
You can watch my performance video on here : https://www.youtube.com/watch?v=q8Q8CCDdSKo
